File: Tarea05/ejercicio9.py
string = str(input("Ingresa una cadena de caracteres para ordenar los caracteres\n"))
lista = []
final = ""
aux = ""
i = 0
while i < len(string):
   lista.append(ord(string[i]))
   i += 1
i = 0
while i < len(lista):
   a = 0
   while i-a >= 0:
      if lista[i] < lista[i-a]:
         aux = lista[i-a]
         lista[i-a] = lista[i]
         lista[i] = aux
         a = 0
      else:
         aux = lista[i]
         lista[i] = lista[i-a]
         lista[i-a] = aux
      a += 1
   i += 1
final = (chr(lista[len(lista)-1]))
i = 0
while i < len(lista)-1:
   final = final + (chr(lista[i]))
   i += 1
print("La cadena ordenada es: %s"%(final))

File: Tarea05/ejercicio9.c
#include <stdio.h>

int main(){
	char car[2000], final[2000];
	int i = 0, longitud, a;
	char aux;
	printf("Ingresa una cadena de caracteres para ordenar los caracteres\n");
	scanf("%s", &car[0]);
	while(car[i] != '\0'){
		i = i +1;
	}
	longitud = i;
	i = 0;
	while(i<longitud){
		a = 0;
		while(i-a>=0){
			if (car[i]< car[i-a]){
				aux = car[i-a];
				car[i-a] = car[i];
				car[i] = aux;
				a = 0;
			}
			else{
				aux = car[i];
				car[i] = car[i-a];
				car[i-a] = aux;
			}
			a = a + 1;
		}
		i = i + 1;
	}
	final[0] = car[longitud-1];
	i = 0;
	while(i<longitud-1){
		final[i+1] = car[i];
		i = i +1;
	}
	printf("%s\n", final);
	return 0;
}